packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -113,13 +109,24 @@ export const AnonymityLevelSlider = ({ className }: AnonymityLevelSliderProps) =
                 <Label>30</Label>
                 <Label>100</Label>
             </LabelsWrapper>
-            {anonymityStatus === AnonymityStatus.Bad && (
-                <WarningWrapper>
-                    <Warning critical withIcon>
-                        <Translation id="TR_ANONYMITY_LEVEL_BAD_WARNING" />
-                    </Warning>
-                </WarningWrapper>
-            )}
+
+            <AnimatePresence initial={!isErrorDisplayed}>
+                {isErrorDisplayed && (
+                    <WarningWrapper
+                        initial={{ height: 0, marginTop: 0, opacity: 0 }}
```

Comment:
nice animation 👍 

nit: It can be defined in object like this for slightly better readability: 
https://github.com/trezor/trezor-suite/blob/feat/remove-anonymity-indicator/packages/suite/src/components/suite/modals/AddAccount/components/MoreCoins.tsx/#L90